sdks/python/apache_beam/runners/interactive/display/display_manager.py

```diff
@@ -147,13 +152,13 @@ def update_display(self, force=False):
         if IPython:
           from IPython.core import display
           display.clear_output(True)
-          rendered_graph = self._renderer.render_pipeline_graph(
-              self._pipeline_graph)
+          rendered_graph = unicode(self._renderer.render_pipeline_graph(
```

Comment:
Hi @cmcneil, thanks for providing this fix!
I'm trying to understand with what content and render option, and using what notebook implementation have you seen this line broken? 
Because we have been using python 3 with interactive beam for the past several months and only encountered rendering issue with Jupyterlab: https://github.com/jupyterlab/jupyterlab/issues/5430 which was a javascript error in the notebook implementation.
Thanks!